.storybook/main.ts

```diff
@@ -1,12 +1,17 @@
 import type { StorybookConfig } from "@storybook/vue3-vite";
-import {globbySync} from "globby"
+// import { globbySync } from "globby";
+
 const config: StorybookConfig = {
-  stories: globbySync(['../stories/*.mdx', `../**/*.stories.@(js|jsx|ts|tsx)`, "!../**/node_modules/**/*"], { cwd: "./.storybook" }),
+  stories: [
+    "../stories/*.mdx",
+    `../**/*.stories.@(js|jsx|ts|tsx)`,
+    "!../**/node_modules/**/*",
+  ],
```

Comment:
why change ?

Reply:
there's no need for globbySync. This has been done internally